hapi-utils/src/test/resources/test.txt

```diff
@@ -0,0 +1 @@
+TEST
```

Comment:
Do we need this added to the repo?

Reply:
I was aware of this. The file was actually generated by a test in the module. So the correct fix is not only by removing it from GitHub, but also to add a clean-up after the mentioned test. Will do in another PR.